apps/studio/components/interfaces/Settings/Addons/Addons.tsx

```diff
@@ -343,6 +345,83 @@ const Addons = () => {
 
           <ScaffoldDivider />
 
+          <ScaffoldContainer>
+            <ScaffoldSection>
+              <ScaffoldSectionDetail>
+                <div className="space-y-6">
+                  <p className="m-0">IPv4</p>
+                  <div className="space-y-2">
+                    <p className="text-sm text-foreground-light m-0">More information</p>
+                    <div>
+                      <Link
+                        href="https://github.com/orgs/supabase/discussions/17817"
+                        target="_blank"
+                        rel="noreferrer"
+                      >
+                        <div className="flex items-center space-x-2 opacity-50 hover:opacity-100 transition">
+                          <p className="text-sm m-0">About IPv4</p>
+                          <IconExternalLink size={16} strokeWidth={1.5} />
+                        </div>
+                      </Link>
+                    </div>
+                  </div>
+                </div>
+              </ScaffoldSectionDetail>
+              <ScaffoldSectionContent>
+                <Alert_Shadcn_ variant="warning">
+                  <IconAlertTriangle className="h-4 w-4" />
+                  <AlertTitle_Shadcn_>
+                    PGBouncer and IPv4 Deprecation on January, 26th
+                  </AlertTitle_Shadcn_>
+                  <AlertDescription_Shadcn_>
+                    Direct connections via db.{projectRef}.supabase.co will resolve to an IPv6
+                    address starting from January 26th. If you plan on not using our connection
+                    pooler and your environment does not support IPv6, consider enabling the add-on.
+                  </AlertDescription_Shadcn_>
```

Comment:
wondering if we should add a learn more CTA here like the one in database settings

![image](https://github.com/supabase/supabase/assets/19742402/f0c17f71-4165-4f5d-bd88-25853c8bcf54)

Reply:
good one